assets/prefabs/gooey.prefab

```diff
@@ -63,6 +63,9 @@
   },
   "Movement": {
     "speed": 3
+  },
+  "GooeyDefence:Health": {
```

Comment:
IMHO use a different name just in case to avoid any chance for name clashing/confusion (and not having to include a namespace like this). In theory this shouldn't be a problem but it would also be very easy to avoid :-)

I don't recall if the use of Gooey as the monster is the target or a placeholder. If it is the target then I'd consider using a literal `GooComponent` to indicate how much goo is left which in turn equates health in this case. Out of goo == death. Could even shrink the model as the goo count lowers

Reply:
I'm still undecided on what to use as the monsters, It'll most likely come down to what models I can use when I implement the different enemies.

Additionally, I still intend for the final module to not depend on core. Currently it's just so that I can easily have an inventory system before I implement the shop system.